shared/src/EMBC.Utilities/Transformation/Transformator.cs

```diff
@@ -25,6 +25,15 @@ public class HbsTransformator : ITransformator
     {
         public async Task<TransformationResult> Transform(TransformationData transformationData)
         {
+            Handlebars.RegisterHelper("customComparerDecimalAmount", (writer, context, parameters) =>
```

Comment:
This helper doesn't make sense to me - the template has one value always null, and there's a row for each type. This helper also has html in it.